File: Java Script - Introduction to Programming/3.1. Simple Conditions/06. Bonus Score.js
function bonusScore(num) {
  num = Number(num)
  let bonus = num
  num <= 100 ? num += 5 : num <= 1000 ? num *= 1.20 : num *= 1.10
  bonus % 2 === 0 ? num++ : bonus % 5 === 0 ? num += 2 : null

  return `${num - bonus}\n${num}`
}

// console.log(bonusScore(20))
// console.log(bonusScore(15875))

File: Pyrhon - Introduction to Programming/3.1. Simple Conditions/06. Bonus Score.py
num = int(input())
bonus = num

if num <= 100: num += 5
elif num <= 1000: num *= 1.20
else: num *= 1.10

if bonus % 2 == 0: num += 1
elif bonus % 5 == 0: num += 2

print(f'{num - bonus}\n{num}')


#20
